scripts/rmc.py

```diff
@@ -45,29 +47,37 @@ def match(self, text):
     def edit_output(self, text):
         return self.edit_fun(text)
 
+
 def is_exe(name):
     from shutil import which
     return which(name) is not None
 
+
 def ensure_dependencies_in_path():
     for program in [RMC_RUSTC_EXE, "symtab2gb", "cbmc", "cbmc-viewer", "goto-instrument", "goto-cc"]:
         ensure(is_exe(program), f"Could not find {program} in PATH")
 
 # Assert a condition holds, or produce a user error message.
+
+
```

Comment:
Why the new whitespace?

Reply:
We check python formatting with `autopep8` on changed files, it inserted the whitepace

Reply:
Ah didn't realize quite how copious it was, might need to move some of these to doc comments to make it happy